### ISSUE TITLE
Miunie does not respond

### ISSUE THREAD

**Issue by petrspelos** [discord-bot-tutorial/Miunie#50]
The latest version is able to log in, but does not respond to any command.

Tested with the profile command.

Tested with a simple echo command.

No command in any Module triggers when sent.

---------

* Please test this as well if it happens to you guys too.
* If someone finds the bug, PR the fix :wink:

**Comment by Charly6596** [comment]
I've checked myself, and the problem is the LanguageResources returning an empty message (should be fixed with #49 and adding a json file with the key:values)
Modified it to return "hello".
Also created a `echo` command, and worked:
```cs
[Command("echo")]
public async Task Echo(CommandContext ctx, [RemainingText]string txt)
{
    await ctx.Channel.SendMessageAsync(txt);
}
```
![image](https://user-images.githubusercontent.com/13708100/52068193-77f18480-257c-11e9-9fd0-a497b94d686c.png)

**Comment by Charly6596** [comment]
Can confirm, merged master with #49, added `PhrasesEn.json` to `Resources/LangResources/PhrasesEn.json` with the following content:
```json
{
  "SHOW_PROFILE": [
    "**YOUR PROFILE**\n{0}",
    "**SECOND PHRASE**\n{0}"
  ]
}
```
![image](https://user-images.githubusercontent.com/13708100/52069609-5a71ea00-257f-11e9-9240-814f6a9773a7.png)

**Comment by petrspelos** [comment]
Fixed with #49
